docker-compose.yml

```diff
@@ -53,6 +53,8 @@ services:
         condition: service_healthy
       agent:
         condition: service_healthy
+      postgres:
+        condition: service_started
```

Comment:
There us an healthcheck on postgres, is there a reason not using  `service_healthy` ?

Reply:
Nope. I think this was set during testing to debug issues. I'll revert this